src/Controller/SiteController.php

```diff
@@ -81,4 +81,12 @@ public function privacy(): Response
     {
         return $this->render('site/privacy.html.twig');
     }
+
+    /**
+     * @Route("/elaomojis", name="elaomojis")
+     */
```

Comment:
Lors du rebase sur master:

```suggestion
    #[Route('/elaomojis', name: 'elaomojis')]
```